libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1593,7 +1597,9 @@ bool ExpressionCompiler::visit(MemberAccess const& _memberAccess)
 		else if (member == "creationCode" || member == "runtimeCode")
 		{
 			TypePointer arg = dynamic_cast<MagicType const&>(*_memberAccess.expression().annotation().type).typeArgument();
-			ContractDefinition const& contract = dynamic_cast<ContractType const&>(*arg).contractDefinition();
+			auto contractType = dynamic_cast<ContractType const*>(arg);
```

Comment:
```suggestion
			auto const& contractType = dynamic_cast<ContractType const&>(*arg);
```

Reply:
Pointer is being used without checking for nullptr, so suggesting to change to reference